packages/babel-generator/test/index.js

```diff
@@ -293,6 +293,18 @@ describe("whitespace", function () {
   });
 });
 
+describe("CodeGenerator", function () {
+  it("constructor", function () {
+    const codeGen = new CodeGenerator(t.stringLiteral("string"));
+    assert(codeGen instanceof CodeGenerator);
+  });
```

Comment:
I don't think this test is very useful. The constructor is also called with the test just above.